index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <section id="welcome-section">
+      <h1>This is a section</h1>
+    </section>
```

Comment:
Please use semantic tags for this section. Semantic tags make your code cleaner and easy to read.